fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -1066,11 +1072,16 @@ private FHIRPersistenceNotSupportedException buildNotSupportedException(String m
                     .text(string("Invalid page size: " + pageSize))
                     .build())
                 .build());
-            pagingContext.setPageSize(10);
+            // Pick a valid default if lenient
+            if (pagingContext.isLenient()) {
+                pagingContext.setPageSize(10);
```

Comment:
Isn't this value already a constant somewhere?

Reply:
There is a constant in FHIRPagingContextImpl, but that is not visible here.  I can create a constant for this.